deployment/frontend/src/components/dashboard/notification/NotificationCard.tsx

```diff
@@ -1,12 +1,13 @@
 import React from 'react'
 import Row from '../_shared/Row'
-import RowProfile from '../_shared/RowProfile'
-import type { IRowProfile } from '../_shared/RowProfile'
-import { DefaultTooltip } from '@/components/_shared/Tooltip'
 import { NotificationType } from '@/schema/notification.schema'
 import Image from 'next/image'
 import Link from 'next/link'
+import dynamic from 'next/dynamic';
 
+const DefaultTooltip = dynamic(() => import('@/components/_shared/Tooltip'), {
```

Comment:
you sure this is not going to break anything? didnt know this tooltip was so expensive